packages/core/src/components/dropdown/test/multiselect/dropdown.e2e.ts

```diff
@@ -0,0 +1,105 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/dropdown/test/multiselect/index.html';
+
+test.describe('tds-dropdown-multiselect', () => {
+  test('renders multiselect dropdown correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const dropdown = page.getByTestId('tds-dropdown-testid');
+    await expect(dropdown).toHaveCount(1);
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('clicking the dropdown opens the dropdown-list, then click Option 1, should not close it', async ({
+    page,
+  }) => {
+    await page.goto(componentTestPath);
+
+    /* click the dropdown button */
+    const dropdownButton = page.getByRole('button', { name: 'Placeholder' });
+    await dropdownButton.click();
+
+    /* Click the Option 1 button */
+    const dropdownListElementOneButton = page
+      .locator('tds-dropdown-option')
+      .filter({ hasText: /Option 1/ });
+    await dropdownListElementOneButton.click();
+
+    await expect(dropdownListElementOneButton).toBeVisible();
+    const dropdownButtonWithOption1 = page.getByRole('button', { name: 'Option 1' });
+    await expect(dropdownButtonWithOption1.first()).toBeVisible();
+
+    /* also check screenshot diff to make sure it says Option 1 */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('clicking the dropdown opens the dropdown-list, then click on all the options', async ({
+    page,
+  }) => {
+    await page.goto(componentTestPath);
+
+    /* click the button */
+    const placeholderElement = page.getByText(/Placeholder/);
+    await placeholderElement.click();
+
+    /* get all checkboxes */
+    const dropdownListElementOneButton = page
+      .locator('tds-dropdown-option')
+      .filter({ hasText: /Option 1/ })
+      .locator('tds-checkbox');
```

Comment:
Same thing here as what I mentioned with the tds-icon in the error text actually. Maybe we could just remove the .locator('tds-checkbox') and it would still work right?

Reply:
ah you're looking for the checked attribute 🤔

Reply:
Hmm, there is a ".toBeChecked()" might be better (assuming it works with our component)
https://playwright.dev/docs/input#checkboxes-and-radio-buttons

Reply:
I guess this might be impossible if we haven't properly made it accessible

Reply:
I can check on it

Reply:
[3537a14](https://github.com/scania-digital-design-system/tegel/pull/435/commits/3537a140bde6204b64b5dbfdfdd1c48635d66009)
Found this solution, what do you think? there are 4 places that updates when something gets checked.

tds-dropdown-option get aria-selected
tds-checkbox gets the checked
input gets the aria-checked
and the dropdown value becomes updates with all checked values...

Reply:
Nice, I think that works. I believe it always needs the aria-selected attribute since they can never be just native checkboxes.
Lets go with this if you agree 👍

Reply:
cool sounds cool with me.